tools/proto2jsonschema/postgen.js

```diff
@@ -38,6 +38,11 @@ console.log(`Fixing paths in dir: ${ROOT}`);
     
     // Create the modified JSON schema output
     const json = JSON.parse(fs.readFileSync(filePath).toString());
+
+    // Replace backslashes with forward-slashes to allow the script
+    // to work on Windows.
+    filePath = filePath.replace(/\\/g, '/');
```

Comment:
Great.

I think we want to normalize the file path right when we get it.

Can we do the following:

```js
filePaths.map(filePath => {
  // Normalize path slashes (`/`, `\`) to allow the script to work on Windows.
  filePath = path.normalize(filePath);
```

This replace may work just as well if you prefer it.

https://nodejs.org/api/path.html#path_path_normalize_path

Reply:
I'll give it a try.

Reply:
Nope, that doesn't work - because it still leaves backslashes as backslashes, and later code assumes it's got forward slashes. (Basically it normalizes to "the platform default", rather than normalizing to forward slashes.)

I suspect if we used path operations everywhere instead of using string operations, we'd be okay - but that would be a bigger change.

Reply:
Alright. I was hoping there's a way to fix the slashes without character replacements, but this seems fine.